### PR TITLE
Reverse columns on "Let's get started" screen

### LINKED ISSUE #8441: No repositories design revisions
entry-point-iteration

Things to note:
* reverse columns in layout (for all cases)
* refine wording on "looking for repositories" in right column
* move “Pro Tip” to the other side (which is not reflected in the above mockup)

### PR DESCRIPTION
**Pull request by kuychaco**
🌵🌵 **Should be merged after #8434, as this PR depends on that renaming work** 🌵 🌵 

Closes #8441

## Description

* Reverse columns on "Let's get started!" screen to reduce friction towards taking the next step.
* Took a pass at improving the filter list's text (placeholder text for text box, and loading text for list).

#### Before: 

GitHub_Desktop-dev

#### After:

GitHub_Desktop-dev

## Release notes

Notes: [Improved] Prominence of "Start tutorial" button on "No Repositories" view

### CODE REVIEW

**Comment by billygriffin** [comment]
Thanks @kuychaco (and @tierninho for testing)! 

I want to make sure two things:
1. How it looks/behaves when you select a repo in the dropdown area is still good/consistent, because that's going to be the path a large number of existing GitHub users will use.
2. Everything still functions well at the minimum window size.

**Comment by outofambit** [comment]
@billygriffin verified!

(the white blocks where text should be are redacted repo names)

Screen Shot 2019-10-14 at 2 37 54 PM
Screen Shot 2019-10-14 at 2 37 37 PM

File: app/src/ui/clone-repository/cloneable-repository-filter-list.tsx [hunk 1]
```diff
@@ -176,6 +176,7 @@ export class CloneableRepositoryFilterList extends React.PureComponent<
         renderNoItems={this.renderNoItems}
         renderPostFilter={this.renderPostFilter}
         onItemClick={this.props.onItemClicked ? this.onItemClick : undefined}
+        placeholderText="Filter repositories"
```
Comment (ampinsk) [inline app/src/ui/clone-repository/cloneable-repository-filter-list.tsx]: Would this be accurate?
```suggestion
        placeholderText="Filter your repositories"
```

File: app/src/ui/clone-repository/cloneable-repository-filter-list.tsx [hunk 2]
```diff
@@ -266,18 +273,13 @@ export class CloneableRepositoryFilterList extends React.PureComponent<
...
-    const endpointName =
-      this.props.account.endpoint === getDotComAPIEndpoint()
-        ? 'GitHub.com'
-        : getHTMLURL(this.props.account.endpoint)
-
     return (
       <div className="no-items empty-repository-list">
         <div>
-          Couldn't find any repositories for the account{' '}
+          Couldn't find any repositories for{' '}
```
Comment (ampinsk) [inline app/src/ui/clone-repository/cloneable-repository-filter-list.tsx]: I'd like to frame this less as an error state if possible. Just a thought: 
```suggestion
          Looks like there are no repositories for{' '}
```

**Comment by kuychaco** [comment]
> @kuychaco Seems like merging #8434 caused some conflicts here, mind taking a look at them?

Resolved. Strangely, when I merged development locally no merge conflicts showed up... I suspect dotcom incorrectly detected conflicts ¯\_(ツ)_/¯

**Comment by niik** [comment]
:sparkles:

**Pull request merged**